artifactory/services/utils/deleteutils.go

```diff
@@ -113,13 +113,13 @@ func FilterCandidateToBeDeleted(deleteCandidates *content.ContentReader, resultW
 				if err != nil {
 					return nil, err
 				}
-				dirsToBeDeleted = append(dirsToBeDeleted, sortedCandidateDirsFile)
+				toBeDeleted = append(toBeDeleted, sortedCandidateDirsFile)
 				// Init buffer.
 				paths = make(map[string]ResultItem)
 				pathsKeys = make([]string, 0, utils.MaxBufferSize)
 			}
 		} else {
-			// Write none dir results.
+			// Write none requested type results.
```

Comment:
Write none requested type results. -->
Write none results of the requested type.